### PR TITLE
sql: add sql statements and document fields of TABLE_STORAGE_STATS

### PR DESCRIPTION
**Pull request by shichun-0415**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
To add four sql statemetns that are missing in the doc.
close the following three issues:
- https://github.com/pingcap/docs/issues/2639
- https://github.com/pingcap/docs-cn/issues/12254
- https://github.com/pingcap/docs-cn/issues/10751

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.6 (TiDB 6.6 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [x] v6.4 (TiDB 6.4 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [x] v5.4 (TiDB 5.4 versions)
- [x] v5.3 (TiDB 5.3 versions)
- [x] v5.2 (TiDB 5.2 versions)
- [x] v5.1 (TiDB 5.1 versions)
- [x] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: information-schema/information-schema-table-storage-stats.md [hunk 1]
```diff
@@ -50,3 +50,14 @@ EMPTY_REGION_COUNT: 1
...
+`TABLE_STORAGE_STATS` 表各列字段含义如下：
+
+* `TABLE_SCHEMA`：
+* `TABLE_NAME`：
+* `TABLE_ID`：
+* `PEER_COUNT`：
+* `REGION_COUNT`：
+* `EMPTY_REGION_COUNT`：
+* `TABLE_SIZE`：
+* `TABLE_KEYS`：
```
Comment (xiongjiwei) [inline information-schema/information-schema-table-storage-stats.md:63]:
```suggestion
* `TABLE_SCHEMA`：表所在 SCHEMA 名字
* `TABLE_NAME`：表名
* `TABLE_ID`：表 ID
* `PEER_COUNT`：副本个数
* `REGION_COUNT`：所在的 REGION 个数
* `EMPTY_REGION_COUNT`：没有该表数据的 REGION 个数
* `TABLE_SIZE`：数据量大小
* `TABLE_KEYS`：表记录个数
```

File: sql-statements/sql-statement-admin-cleanup.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+---
+title: ADMIN CLEANUP
```
Comment (TomShawn) [inline sql-statements/sql-statement-admin-cleanup.md:2]: already added in https://github.com/pingcap/docs-cn/pull/13209

File: sql-statements/sql-statement-admin-recover.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+---
+title: ADMIN RECOVER INDEX
```
Comment (TomShawn) [inline sql-statements/sql-statement-admin-recover.md:2]: already added in https://github.com/pingcap/docs-cn/pull/13209

File: information-schema/information-schema-table-storage-stats.md [hunk 4]
```diff
@@ -50,3 +50,14 @@ EMPTY_REGION_COUNT: 1
         TABLE_KEYS: 0
 1 row in set (0.00 sec)
 ```
+
+`TABLE_STORAGE_STATS` 表各列字段含义如下：
+
+* `TABLE_SCHEMA`：表所在 SCHEMA 名字
```
Comment (Oreoxmt) [inline information-schema/information-schema-table-storage-stats.md:56]:
```suggestion
* `TABLE_SCHEMA`：表所在 schema 名字
```

File: information-schema/information-schema-table-storage-stats.md [hunk 5]
```diff
@@ -50,3 +50,14 @@ EMPTY_REGION_COUNT: 1
...
+* `TABLE_NAME`：表名
+* `TABLE_ID`：表 ID
+* `PEER_COUNT`：副本个数
+* `REGION_COUNT`：所在的 REGION 个数
```
Comment (Oreoxmt) [inline information-schema/information-schema-table-storage-stats.md:60]:
```suggestion
* `REGION_COUNT`：表所在的 Region 个数
```

File: information-schema/information-schema-table-storage-stats.md [hunk 6]
```diff
@@ -50,3 +50,14 @@ EMPTY_REGION_COUNT: 1
...
+* `EMPTY_REGION_COUNT`：没有该表数据的 REGION 个数
```
Comment (Oreoxmt) [inline information-schema/information-schema-table-storage-stats.md:61]:
```suggestion
* `EMPTY_REGION_COUNT`：没有该表数据的 Region 个数
```

**Review by Oreoxmt — APPROVED** [review]
Rest LGTM

File: information-schema/information-schema-table-storage-stats.md [hunk 7]
```diff
@@ -50,3 +50,14 @@ EMPTY_REGION_COUNT: 1
...
+
+`TABLE_STORAGE_STATS` 表各列字段含义如下：
+
+* `TABLE_SCHEMA`：表所在 schema 名字
+* `TABLE_NAME`：表名
+* `TABLE_ID`：表 ID
+* `PEER_COUNT`：副本个数
+* `REGION_COUNT`：表所在的 Region 个数
+* `EMPTY_REGION_COUNT`：没有该表数据的 Region 个数
+* `TABLE_SIZE`：数据量大小
```
Comment (shichun-0415) [inline information-schema/information-schema-table-storage-stats.md:62]:
```suggestion
* `TABLE_SIZE`：数据量大小，单位 MiB
```

File: sql-statements/sql-statement-admin-cleanup.md [hunk 8]
```diff
@@ -0,0 +1,74 @@
...
```
Comment (Oreoxmt) [inline sql-statements/sql-statement-admin-cleanup.md:2]:
```suggestion
title: ADMIN CLEANUP INDEX
```
title 和 H1 要保持一致

File: sql-statements/sql-statement-admin-recover.md [hunk 9]
```diff
@@ -0,0 +1,72 @@
...
+```
+
+> **注意：**
+>
+> 当由于副本丢失导致数据索引出现不一致时：
+>
+> - 通常行数据与索引数据都有丢失，此时需要同时使用 [`ADMIN CLEANUP INDEX`](/sql-statements/sql-statement-admin-cleanup.md) 与 `ADMIN RECOVER INDEX` 语句保证行数据与索引数据的一致性。
+> - `ADMIN RECOVER INDEX` 总是单并发执行。在表数据量大时，建议通过重建索引的方式进行索引数据的恢复。
+> - `ADMIN RECOVER INDEX` 在执行期间，不会为对应的表或索引记录加锁，这意味着 TiDB 允许同时有其他的会话对表记录进行更新或修改。然而，在此情况下 `ADMIN RECOVER INDEX` 可能无法正确处理所有数据。因此，在执行 `ADMIN RECOVER INDEX` 时，请避免同时修改表数据。
+> - 若你是 PingCAP 商业用户，此时建议联系 PingCAP 支持工程师进行处理。
```
Comment (Oreoxmt) [inline sql-statements/sql-statement-admin-recover.md:61]:
```suggestion
> - 若你使用 TiDB 企业版，此时建议[提交工单](https://support.pingcap.cn/)联系 PingCAP 支持工程师进行处理。
```
https://docs.pingcap.com/zh/tidb/stable/support

**Review by Oreoxmt — APPROVED** [review]
Rest LGTM

File: sql-statements/sql-statement-admin-cleanup.md [hunk 10]
```diff
@@ -0,0 +1,74 @@
...
+```
+
+> **注意：**
+>
+> 当由于副本丢失导致数据索引出现不一致时：
+>
+> - 通常行数据与索引数据都有丢失，此时需要同时使用 `ADMIN CLEANUP INDEX` 与 [`ADMIN RECOVER INDEX`](/sql-statements/sql-statement-admin-recover.md) 语句保证行数据与索引数据的一致性。
+> - `ADMIN CLEANUP INDEX` 总是单并发执行。在表数据量大时，建议通过重建索引的方式进行索引数据的恢复。
+> - `ADMIN CLEANUP INDEX` 在执行期间，不会为对应的表或索引记录加锁，TiDB 允许其他的会话同时对表记录进行更新或修改。然而，在此情况下，`ADMIN CLEANUP INDEX` 可能无法正确处理所有表记录。因此，在执行 `ADMIN CLEANUP INDEX` 时，请避免同时修改表数据。
+> - 若你是 PingCAP 商业用户，此时建议联系 PingCAP 支持工程师进行处理。
```
Comment (shichun-0415) [inline sql-statements/sql-statement-admin-cleanup.md:63]:
```suggestion
> - 若你使用 TiDB 企业版，此时建议[提交工单](https://support.pingcap.cn/)联系 PingCAP 支持工程师进行处理。
```

**Review by bb7133 — APPROVED** [review]
LGTM

**Pull request merged**